spec/requests/transition_checker_email_subscription_spec.rb

```diff
@@ -125,28 +114,14 @@
     context "when the user doesn't have a high enough level of authentication" do
       let(:session_identifier) { placeholder_govuk_account_session(level_of_authentication: "level-1") }
 
-      it "calls the account manager" do
-        stub = stub_request(:post, "#{Plek.find('account-manager')}/api/v1/transition-checker/email-subscription")
-          .with(body: hash_including(topic_slug: "slug"))
-          .to_return(status: 200)
-
+      it "returns a 403 and the required level" do
```

Comment:
do we need to updat the [docs](https://github.com/alphagov/account-api/blob/main/docs/api.md#post-apitransition-checker-email-subscription) too?